specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2018-12-20/bms.json

```diff
@@ -0,0 +1,3603 @@
+          }
+        }
+      }
+    },
+    "/Subscriptions/{subscriptionId}/providers/Microsoft.RecoveryServices/locations/{azureRegion}/backupCrrJob":{
+      "post":{
+        "tags":[
+          "CrrJobDetails"
+        ],
+        "summary":"Get CRR job details from target region.",
+        "operationId":"BackupCrrJobDetails_Get",
+        "produces":[
+          "application/json"
+        ],
+        "parameters":[
+          {
+            "$ref":"#/parameters/ApiVersion"
+          },
+          {
+            "$ref":"#/parameters/AzureRegion"
+          },
+          {
+            "$ref":"#/parameters/SubscriptionId"
+          }
+        ],
+        "responses":{
+          "200":{
+            "description":"OK",
+            "schema":{
+              "$ref":"#/definitions/JobResource"
+            }
+          }
+        },
+        "x-ms-examples":{
+          "Get Cross Region Restore Job Details in the secondary region":{
+            "$ref":"./examples/AzureIaasVm/GetCrrJob_Post.json"
+          }
+        }
+      }
+    },
+    "/Subscriptions/{subscriptionId}/providers/Microsoft.RecoveryServices/locations/{azureRegion}/backupCrrJobs":{
+      "post":{
+        "tags":[
+          "BackupCrrJobs"
+        ],
+        "summary":"Gets the list of CRR jobs from the target region.",
+        "operationId":"BackupCrrJobs_List",
+        "produces":[
+          "application/json"
+        ],
+        "parameters":[
+          {
+            "$ref":"#/parameters/ApiVersion"
+          },
+          {
+            "$ref":"#/parameters/AzureRegion"
+          },
+          {
+            "$ref":"#/parameters/SubscriptionId"
+          }
+        ],
+        "responses":{
+          "200":{
+            "description":"OK",
+            "schema":{
+              "$ref":"#/definitions/JobResourceList"
+            }
+          }
+        },
+        "x-ms-pageable":{
+          "nextLinkName":"nextLink"
+        },
+        "x-ms-examples":{
+          "List Cross Region Restore Jobs in the secondary region":{
+            "$ref":"./examples/AzureIaasVm/ListCrrJobs_Post.json"
+          }
+        }
+      }
+    },
+    "/Subscriptions/{subscriptionId}/providers/Microsoft.RecoveryServices/locations/{azureRegion}/backupCrrOperationResults/{operationId}":{
+      "get":{
+        "tags":[
+          "CrrOperationResults"
+        ],
+        "operationId":"CrrOperationResults_Get",
+        "produces":[
+          "application/json"
+        ],
+        "parameters":[
+          {
+            "$ref":"#/parameters/ApiVersion"
+          },
+          {
+            "$ref":"#/parameters/AzureRegion"
+          },
+          {
+            "$ref":"#/parameters/SubscriptionId"
+          },
+          {
+            "name":"operationId",
+            "in":"path",
+            "required":true,
+            "type":"string"
+          }
+        ],
+        "responses":{
```

Comment:
This is a GET however neither of the responses return a schema.  Is that expected?

Reply:
Ping @gaalla

Reply:
Yes, this is the expected behavior.